docs/sources/flow/reference/config-blocks/http.md

```diff
@@ -150,3 +156,51 @@ The following versions are recognized:
 * `TLS12` for TLS 1.2
 * `TLS11` for TLS 1.1
 * `TLS10` for TLS 1.0
+
+
+### windows certificate filter block
+
+The `windows_certificate_filter` block is used to configure retrieving certificates from the built-in Windows
+certificate store. When using the `windows_certificate_filter` block
+the following TLS settings are overridden and will cause an error if defined.
+
+* `cert_pem`
+* `cert_file`
+* `key_pem`
+* `key_file`
+* `client_ca`
+* `client_ca_file`
+
+{{% admonition type="warning" %}}
+This feature is not available on any platforms other than Windows.
+
+TLS min and max may not be compatible with the certificate stored in the Windows certificate store. The `windows_certificate_filter`
+will serve the found certificate even if it is not compatible with the specified TLS version.
+{{% /admonition %}}
+
+
+### server block
+
+The `server` block is used to find the certificate to check the signer. If multiple certificates are found the 
+`windows_certificate_filter` will choose the certificate with the expiration farthest in the future.
+
+Name | Type           | Description                                                                               | Default | Required
+---- |----------------|-------------------------------------------------------------------------------------------|---------| --------
+`store` | `string`       | Name of the system store to look for the server Certificate ex LocalMachine, CurrentUser. | `""`    | yes
+`system_store` | `string`       | Name of the store to look for the server Certificate ex My, CA.                           | `""`    | yes
```

Comment:
```suggestion
`store` | `string`       | Name of the system store to look for the server Certificate, for example, LocalMachine, CurrentUser. | `""`    | yes
`system_store` | `string`       | Name of the store to look for the server Certificate, for example, My, CA.                           | `""`    | yes
```

Do we want/need to include examples here? Generally these are noted below the table.